src/sql/base/browser/ui/radioButton/radioButton.ts

```diff
@@ -74,7 +74,7 @@ export class RadioButton extends Widget {
 	}
 
 	public set checked(val: boolean) {
-		if (this.inputElement.checked !== this._internalCheckedStateTracker) {
+		if (val !== this._internalCheckedStateTracker) {
```

Comment:
So I think we have another bug here too - it doesn't look like this is updating the internal state when the button is clicked if it's not part of a group. We have the click handler but that only fires off the change event if the button has a name - we should always be firing the change event for the button itself regardless.